recipes-apps/ovmenu-ng-skripts/ovmenu-ng-skripts_0.1.bb

```diff
@@ -19,7 +19,8 @@ SRC_URI =      "\
 	file://download-igc.sh \
 	file://upload-all.sh \
 	file://upload-xcsoar.sh \
-	file://download-xcsoar.sh \
+	file://download-all.sh \

```

Comment:
Why the rename?

The recipe in the current form fails to build, because `download-xcsoar.sh` is still referenced in this file below.

Reply:
This was renamed because the script was misnamed since the previous bild system. The feature never worked because it looked for "donwload.xcsoar" while the script was named "download-all"